src/utils.ts

```diff
@@ -50,14 +52,20 @@ export const getShouldAlertAboutPartyHalfWay = async (
     return false;
   }
 
-  // todo:
-  // 1. create api endpoint that does the following:
-  //   * if partybid - fetch auction reserve price
-  //   * if partybuy - fetch maxPrice from contract
-  //   * if collectionbuy - fetch current floor price
-  // 2. compare (fetched value) / totalContributions >= .5
-  // 3. return boolean
-  const isPartyHalfway = false;
+  const API_ENDPOINT = ""; // todo: add this
```

Comment:
let me know if we should check in this api endpoint, or if it makes more sense to use an env var